tests/autocompletion/Earthfile

```diff
@@ -21,6 +21,7 @@ config
 ls 
 org 
 prune 
+satellite 
```

Comment:
This feels wrong: it seems like there's a bug in the autocompletion logic -- if you want to punt on this issue can you create an issue to track it?

Reply:
before you do that can you try changing https://github.com/earthly/earthly/blob/main/autocomplete/complete.go#L344

`subCommands := app.Commands` to `subCommands = getVisibleCommands(app.Commands)`

Reply:
Yeah I was a bit surprised by the fact it showed up while marked as hidden. Happy to try the fix.